wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_column_header.html

```diff
@@ -0,0 +1,30 @@
+{% extends "wagtailadmin/tables/column_header.html" %}
+{% load wagtailadmin_tags i18n %}
+
+{% block after_label %}
+    {% if page_obj %}
+        {% with start_index=page_obj.start_index end_index=page_obj.end_index result_count=page_obj.paginator.count %}
+            {% if is_searching %}
+                {% if is_searching_whole_tree %}
+                    {% blocktranslate trimmed %}
```

Comment:
Nitpick, just noticed that we use both `blocktranslate` and `blocktrans` here. I don't really mind but we might want to be more consistent between the two.